CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Changed
+- Search - In a request with no smart fields, do not return fields that are hidden from the UI.
```

Comment:
"Search" is a bit misleading. This change is not related to the "search" feature.
It is about performance optimization.